framework/contracts/account/tests/create.rs

```diff
@@ -78,9 +78,9 @@ fn create_one_account() -> AResult {
 
     assert_that!(&vc_config).is_equal_to(&expected);
 
-    let account_list = registry.account(TEST_ACCOUNT_ID)?;
+    let account_list = registry.get_account(TEST_ACCOUNT_ID)?;
```

Comment:
Are we keeping `get_` prefix for the interface?

Reply:
we could do just `account` as we changed the name of the actual query to `accounts`

Reply:
renamed it